classes/web_processor.php

```diff
@@ -33,15 +33,29 @@ class web_processor implements processor {
      */
     protected $profile;
 
+    /**
+     * @var sample_set
+     */
```

Comment:
```suggestion
    /** @var sample_set */
```
Can be a one liner, same for similar class props